sim_city.py

```diff
@@ -0,0 +1,116 @@
+class City:
+    """
+    Represents a city in Ackland, should help calculate the tax of each city
+    """
+
+    def __init__(self):
+        self.neighborhoods = {}
+        self.base_tax = 1000
```

Comment:
Usually when there's a constant number involved, it's better to use a default constant we can easily change.
So here for example, you can use:
```
DEFAULT_BASE_TAX = 1000

def __init__(self, base_tax=DEFAULT_BASE_TAX):
    self.neighborhoods = {}
    self.base_tax = base_tax
```